vhost_rs/src/vhost_user/mod.rs

```diff
@@ -41,6 +41,9 @@ pub use self::slave::SlaveListener;
 mod slave_req_handler;
```

Comment:
nit: fix the capitalisation in your commit messages (don't capitalise the first letter of your first line of your commit message.)

Reply:
OK.